### PR TITLE
Linked Contributing Guidelines

### PR DESCRIPTION
**Pull request by imanishbarnwal**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/docs/blob/master/CONTRIBUTING.md) before creating this PR.

##### Description of change
Linked CONTRIBUTING.md in README.md
Issue: #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ https://www.loginradius.com/docs/developer
 - `npm start` in this directory
 -  Open a browser to the link provided in the console.
 
-
+**Note:** Before start contributing to this docs you must read [contributing guidelines](CONTRIBUTING.md).
```
Comment (NauhcNoohc) [inline README.md:17]: I think the small letter `md` would not link to the file. I was tried to click the link in display mode and it bring me to the 404 page.

https://github.com/LoginRadius/docs/blob/master/CONTRIBUTING.MD

```suggestion
**Note:** Before start contributing to this docs you must read [contributing guidelines](CONTRIBUTING.MD).
```

Comment (imanishbarnwal) [reply README.md]: Thanks for letting me know on this. I just updated my PR kindly check now.

**Comment by aman-agrwl** [comment]
@imanishbarnwal 

Thanks for contributing to the repo. We can surely improve as per your suggestions. But as per the HacktoberFest guideline, we will not be accepting minor text fixes as a PR. Please suggest and improve some quantifiable changes in the pull request to make your PR eligible for HacktoberFest.

**Comment by imanishbarnwal** [comment]
> @imanishbarnwal 
> 
> Thanks for contributing to the repo. We can surely improve as per your suggestions. But as per the HacktoberFest guideline, we will not be accepting minor text fixes as a PR. Please suggest and improve some quantifiable changes in the pull request to make your PR eligible for HacktoberFest.

Okay, no problem is there any valid issue in this doc repo. I think this repo is only for documentation purpose that's why I added that.

**Comment by vyasriday** [comment]
@imanishbarnwal Thanks for contributing.  We discourage small fixes as it brings spam. But considering this adds value to the repo we will consider it. Please update it as per our this https://github.com/LoginRadius/engineering-portal

**Comment by imanishbarnwal** [comment]
> @imanishbarnwal Thanks for contributing. We discourage small fixes as it brings spam. But considering this adds value to the repo we will consider it. Please update it as per this https://github.com/LoginRadius/engineering-portal#how-to-contribute

Okay cool. I am adding it dev branch.

**Comment by vyasriday** [comment]
@imanishbarnwal Why have you created 2 PRs with same changes?

**Comment by imanishbarnwal** [comment]
> @imanishbarnwal Why have you created 2 PRs with same changes?

As the first one is not on dev branch... You can close first one.

**Comment by vyasriday** [comment]
But we accept on the master branch. I did not ask you to create on dev. What I asked was to follow the same pattern for adding the contributing link like it was in the repo I linked to. https://github.com/LoginRadius/engineering-portal#how-to-contribute 
Here we have a heading 
### How to Contribute 
and then the link. So follow that. I am closing #59

**Comment by imanishbarnwal** [comment]
> But we accept on the master branch. I did not ask you to create on dev. What I asked was to follow the same pattern for adding the contributing link like it was in the repo I linked to. https://github.com/LoginRadius/engineering-portal#how-to-contribute
> Here we have a heading
> 
> ### How to Contribute
> and then the link. So follow that. I am closing #59
> 
> Looks like I wasn't clear in my comment. Sorry for that.

Okay, I just corrected and raised a PR check now.

File: README.md [hunk 2]
```diff
@@ -15,3 +15,8 @@ https://www.loginradius.com/docs/developer
 -  Open a browser to the link provided in the console.
 
 
+## How to contribute
+
+We appreciate all kinds of contributions from anyone, be it finding an issue or writing a blog.
```
Comment (vyasriday) [inline README.md:20]: Change it to `We appreciate all kinds of contributions from our contributors.`

**Review by vyasriday — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (imanishbarnwal) [reply README.md]: Just updated check now.

**Review by aman-agrwl — APPROVED** [review]

**Review by vyasriday — APPROVED** [review]

**Pull request merged**